hanami-cli.gemspec

```diff
@@ -27,6 +27,8 @@ Gem::Specification.new do |spec|
     f.match(%r{^(test|spec|features)/})
   end
 
+  spec.required_ruby_version = ">= 2.4.0"
```

Comment:
hanami cli works with jruby too. How we can handle it here?

Reply:
That's a good question. I'm trying to get up to speed on jruby support. The other hanami gems seem to use this setting already so I thought it was an oversight here. I'll come back to this after I've completed this process for them.

Reply:
Folks, this is related to jruby 9.1 because it targets ruby 2.3, jruby 9.2 will target ruby 2.5